src/lib/componentloader.js

```diff
@@ -7,29 +7,29 @@ ComponentLoader.prototype = {
   loadComponentsData: function () {
     var xhr = new window.XMLHttpRequest();
     // @todo Remove the sync call and use a callback
-    xhr.open('GET', 'https://raw.githubusercontent.com/aframevr/aframe-components/master/components.json', false);
+    xhr.open('GET', 'https://aframe.io/aframe-registry/build/' + AFRAME.version + '.json');
```

Comment:
I've just added the function `getMajorVersion (version)` that takes cares of that